planet/cli/subscriptions.py

```diff
@@ -0,0 +1,126 @@
+"""Subscriptions CLI"""
+
+import itertools
+import json
+
+import click
+
+import planet
+from planet.cli.cmds import coro, translate_exceptions
+from planet.cli.io import echo_json
+from planet.exceptions import PlanetError
+
+# A collection of fake subscriptions for testing. Tests will
+# monkeypatch this attribute.
+_fake_subs = None
+
+
+# For test use.
+def _count_fake_subs():
+    return len(_fake_subs)
```

Comment:
The `_fake_subs` attribute and `_count_fake_subs` function, at this stage of our work, give us a very rough fake Subscriptions API.